src/features/auth/components/LoginForm/LoginForm.ui.tsx

```diff
@@ -50,12 +56,23 @@ export function LoginFormUI({ form, onSubmit }: LoginFormUIProps) {
           </div>
           <div className="space-y-2">
             <Label htmlFor="password">パスワード</Label>
-            <Input
-              id="password"
-              type="password"
-              {...register('password')}
-              aria-invalid={!!errors.password}
-            />
+            <div className="relative">
+              <Input
+                id="password"
+                type={showPassword ? 'text' : 'password'}
+                {...register('password')}
+                aria-invalid={!!errors.password}
+                className="pr-10"
+              />
+              <button
+                type="button"
+                onClick={onTogglePassword}
+                className="absolute right-3 top-1/2 -translate-y-1/2 text-sm"
+                aria-label={showPassword ? 'パスワードを非表示にする' : 'パスワードを表示する'}
+              >
+                {showPassword ? '🚫👁️' : '👁️'}
+              </button>
```

Comment:
汎用コンポーネントを活用するように修正をお願いします！
今回のケースでは大きな恩恵はありませんが、基本的にはシステム全体でスタイルを統一するために利用することが多いです。

```suggestion
              <Button
                type="button"
                variant="ghost"
                size="icon"
                onClick={onTogglePassword}
                aria-label={
                  showPassword
                    ? 'パスワードを非表示にする'
                    : 'パスワードを表示する'
                }
                className="absolute right-2 top-1/2 -translate-y-1/2 p-0 bg-transparent hover:bg-transparent focus:ring-0"
              >
                {showPassword ? (
                  <EyeOff className="h-4 w-4" />
                ) : (
                  <Eye className="h-4 w-4" />
                )}
              </Button>
```

Reply:
上記のサンプルコードに記載していますが、アイコンは`lucide`を活用するようにしようと思いますので、修正をお願いします！
https://lucide.dev/

次のようにインポートすることで、利用することができるかと思います。

```tsx
import { Eye, EyeOff } from 'lucide-react';
```